### PR TITLE
Testnet docs

### PR DESCRIPTION
**Pull request by bengtlofgren**
Changed to be up to date

### CODE REVIEW

File: documentation/docs/src/testnets/testnets.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+# Namada Testnets
+
+Testnets on Namada serve the purpose of allowing users on the internet to test the latest features of the protocol. Beware of bugs :warning: :ant: ! Pest control can be reached through any of our socials, and we greatly appreciate any bug reports :eyes: :airplane_arriving: 
+
+Testnets are released on a fortnightly cycle (every other Tuesday). Sometimes changes are large, sometimes they are small. The [github changelog](https://github.com/anoma/namada/tree/main/.changelog) will specify exactly what changes have been made in between versions.
```
Comment (cwgoes) [inline documentation/docs/src/testnets/testnets.md:6]:
```suggestion
Testnets are released on a fortnightly cycle (every other Tuesday). Sometimes changes are large, sometimes they are small. The [Github changelog](https://github.com/anoma/namada/tree/main/.changelog) will specify exactly what changes have been made in between versions.
```

File: documentation/docs/src/testnets/testnets.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+The instructions to join the latest running testnet can be found [HERE](https://hackmd.io/WDhESkCeSWepuki1F1scxg)
+
+The block explorer is currently in development. The latest version can be found at [namada.world](https://namada.world/)
+
+We much appreciate any feedback, which we would gladly appreciate through either [twitter](https://twitter.com/namadanetwork), [reddit](https://www.reddit.com/r/namada), or [github](https://github.com/anoma/namada/issues).
```
Comment (cwgoes) [inline documentation/docs/src/testnets/testnets.md:12]:
```suggestion
We much appreciate any feedback, which we would gladly appreciate through either [Twitter](https://twitter.com/namadanetwork), [Reddit](https://www.reddit.com/r/namada), or [Github](https://github.com/anoma/namada/issues). 
```

**Review by cwgoes — APPROVED** [review]
ack modulo nits

**Review by cwgoes — APPROVED** [review]

**Review by awasunyin — APPROVED** [review]
This is an exceptional "yolo" approve - don't do this at home. I'll review the contents properly on docs.namada.net

**Pull request merged**